lib/bridge/MatrixHandler.js

```diff
@@ -203,33 +203,34 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
     if (cmd === "!help") {
         let helpCommands = {
             "!join": {
-                example: `!join irc.example.com #channel [key]`,
+                example: `!join [irc.example.net] #channel [key]`,
                 summary: `Join a channel (with optional channel key)`,
             },
             "!nick": {
-                example: `!nick irc.example.com DesiredNick`,
+                example: `!nick [irc.example.net] DesiredNick`,
                 summary: "Change your nick. If no arguments are supplied, " +
                          "your current nick is shown.",
             },
             "!whois": {
-                example: `!whois NickName|@alice:matrix.org`,
+                example: `!whois [irc.example.net] NickName|@alice:matrix.org`,
                 summary: "Do a /whois lookup. If a Matrix User ID is supplied, " +
                          "return information about that user's IRC connection.",
             },
             "!storepass": {
-                example: `!storepass [irc.example.com] passw0rd`,
+                example: `!storepass [irc.example.net] passw0rd`,
                 summary: `Store a NickServ password (server password)`,
             },
             "!removepass": {
-                example: `!removepass [irc.example.com]`,
+                example: `!removepass [irc.example.net]`,
                 summary: `Remove a previously stored NickServ password`,
             },
             "!quit": {
                 example: `!quit`,
-                summary: `Leave all bridged channels and remove your connection to IRC`,
+                summary: "Leave all bridged channels, on all networks, and remove your " +
+                         "connections to all networks.",
             },
             "!cmd": {
-                example: `!cmd [irc.server] COMMAND [arg0 [arg1 [...]]]`,
+                example: `!cmd [irc.example.net] COMMAND [arg0 [arg1 [...]]]`,
                 summary: `Issue a raw IRC command. These will not produce a reply.`,
```

Comment:
Could you also say that the COMMAND has to be capitalized regardless of IRC generally being a case-insensitive protocol?

Reply:
Done.  I have created an issue for making `!cmd` case insensitive.